docs/kyma/docs/04-05-use-your-own-image.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Use your own image
+type: Installation
+---
+
+You can use your own image to prepare the configuration file when installing Kyma on a GKE or AKS cluster.
+
+>**NOTE:** Follow these steps both for your own and the `xip.io` default domain.
+
+1. Check out [kyma-project](https://github.com/kyma-project/kyma) and enter the root folder.
+
+2. Build an image that is based on the current Installer image and includes the current installation and resources charts. Run:
+
+    ```
+    docker build -t kyma-installer:latest -f tools/kyma-installer/kyma.Dockerfile .
+    ```
+
+3. Push the image to your Docker Hub. Run:
+    ```
+    docker tag kyma-installer:latest {YOUR_DOCKER_LOGIN}/kyma-installer:latest
+    docker push {YOUR_DOCKER_LOGIN}/kyma-installer:latest
+    ```
+
+4. Prepare the deployment file.
```

Comment:
I suggest 4 tabs, as it can be misleading that on tab GKE I use only bullet 1 and 3 for xip and 2 and 4 for own domain

Reply:
shouldn't these be steps instead of bullet

Reply:
Actually, bullet points as all of them are alternatives to one action, not two steps in a sequence.

Reply:
...and I will think of using tabs here